config/environments/production.rb

```diff
@@ -11,17 +11,17 @@
 
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = {
-    address:              'smtp.sendgrid.net',
+    address:              'smtp.mailgun.org',
     port:                 587,
     authentication:       :plain,
-    user_name:            ENV["SENDGRID_USERNAME"],
-    password:             ENV["SENDGRID_PASSWORD"],
-    domain:               'heroku.com',
-    enable_starttls_auto: true 
+    user_name:            ENV["MAILGUN_USERNAME"],
+    password:             ENV["MAILGUN_PASSWORD"],
+    domain:               'sandboxe58ede60e51b49b18eedba8b4c289a2d.mailgun.org',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.